src/ApiClient.php

```diff
@@ -282,6 +292,116 @@ public function createOrReplaceTripPricings(string $rentalUnitId, array $tripPri
             ->getData()->getCreateOrReplaceTripPricings();
     }
 
+    // region Date pricing
+    public function createOrReplaceDatePricings(
+        string $rentalUnitId,
+        DatePricingCollection $datePricingCollection
+    ): DatePricingCollection
+    {
+        $normalizedDatePricingCollection = [];
+        foreach ($datePricingCollection as $datePricing) {
+            if ($datePricing instanceof DatePricing) {
+                $normalizedDatePricingCollection[] = $datePricing->toArray();
+            } elseif (is_array($datePricing)) {
+                $normalizedDatePricingCollection[] = $datePricing;
+            }
+        }
+
+        $mutation = $this->queryBuilder->createCreateOrReplaceDatePricingsMutation();
+        $variables = [
+            'input' => [
+                'rentalUnitId' => $rentalUnitId,
+                'datePricings' => $normalizedDatePricingCollection,
+            ],
+        ];
+        $result = $this->runQuery($mutation, $variables);
+
+        /** @var CreateOrReplaceDatePricingsCallResponseBody $response */
+        $response = $this->parseResult($result, CreateOrReplaceDatePricingsCallResponseBody::class);
+        return $response->getData()->getCreateOrReplaceDatePricings();
```

Comment:
return zou volgesn psr een regel ertussen moeten hebben volgens mij (kan alleen de regel niet meer vinden in de documentatie)